set.mm

```diff
@@ -5631,6 +5662,15 @@ use disjunction (although this is not required since definitions are
   anbi2 $p |- ( ( ph <-> ps ) -> ( ( ch /\ ph ) <-> ( ch /\ ps ) ) ) $=
     ( wb id anbi2d ) ABDZABCGEF $.
 
+  ${
+    anbi1cd.1 $e |- ( ph -> ( ps <-> ch ) ) $.
+    $( Deduction adding a left conjuct to the left side and a right conjunct to
```

Comment:
Introduce a proposition as left conjunct on the left-hand side and right conjunct on the right-hand side of an equivalence. Deduction form.